CHANGELOG.md

```diff
@@ -2,6 +2,52 @@
 
 ## [Unreleased]
 
+## [1.5.0] - 2023-03-30
+
+### Configuration
+
+#### Bugfixes
+
+- Remove `SPLUNK_METRICS_ENDPOINT` from Instrumentation Libraries
+  (it was never really stable).
+
+#### Enhancements
+
+- Add `SPLUNK_PROFILER_MEMORY_ENABLED`.
+- Deprecate `jaeger-thrift-splunk` option for `OTEL_TRACES_EXPORTER`.
+- Remove the policy regarding Zipkin exporter.
+- OTLP exporter can use either `grpc` or `http/protobuf`
+  as the default transport protocol.
+
+### Repository
+
+#### Enhancements
+
+- Add tag protection rule requirement.
+- Add Dependabot security configuration requirements.
+- Grant Admin role for maintainers team.
+- Refine documentation requirements.
```

Comment:
This is pretty vague. Can we be more specific about what GDI repositories need to change or be aware of here?

Reply:
PTAL https://github.com/signalfx/gdi-specification/pull/226/commits/387812410a06f98c9561a1b1a554808b4acf8d14